### PR TITLE
Support more options for schema generation

### PR DESCRIPTION
**Pull request by russgold**
With this change, it is now possible to configure the plugin to:

- disable the "additionalProperties" field (required for K8s CRD validation)
- generate object fields in-line rather than using the $ref field (required for K8s CRD validation)
- specify the name and location of the file to generate (needed to generate one for validation and one for the website)

### CODE REVIEW

File: model/src/main/java/oracle/kubernetes/weblogic/domain/v2/ProbeTuning.java [hunk 1]
```diff
@@ -24,6 +24,12 @@
 
   public ProbeTuning() {}
 
+  void copyValues(ProbeTuning fromProbe) {
```
Comment (rjeberhard) [inline model/src/main/java/oracle/kubernetes/weblogic/domain/v2/ProbeTuning.java]: Any idea why these changes are here?  Otherwise, everything looks good.

**Comment by russgold** [comment]
That’s was a minor refactoring that I forgot to put into the previous change. I had in mind collecting a bunch of such cleanups, but didn’t wind up doing any others.

**Comment by russgold** [comment]
There is, BTW, a corresponding change in ServerPod
